### ISSUE TITLE
Non-existent files suddenly appeared in Explorer

### ISSUE THREAD

**Issue by 5wattbulb** [microsoft/vscode#145676]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:
- OS Version:

Steps to Reproduce:

1.No idea. They just suddenly appeared. They have weird UTF symbol names, they can't be viewed in the editor (error message = doesn't exist) and, if I click to delete them I get 'file doesn't exist and so can't be deleted'. They also appear in Windows Explorer in the Linux/Manjaro/home/tom folder and can't be deleted there either because they don't exist (they all show as 0 KB in size). There were others with similar weird names that I was able to open in the editor (blank) and delete.   
2.
![image](https://user-images.githubusercontent.com/76473295/159472876-6284b384-2fb9-4d58-9b8b-314d2e4cffc3.png)

**Comment by JacksonKearl** [comment]
No link at the moment but Ive seen reports where this was caused by a prerelease version of the C++ extension, could you check if downgrading that helps?

**Comment by 5wattbulb** [comment]
> No link at the moment but Ive seen reports where this was caused by a prerelease version of the C++ extension, could you check if downgrading that helps?

Thanks for the reply. I disabled all the C++ extensions that were installed and reloaded but no difference.

**Comment by JacksonKearl** [comment]
> They also appear in Windows Explorer in the Linux/Manjaro/home/tom folder and can't be deleted there either because they don't exist

This seems to be a system issue in that case? Not specific to the VS Code explorer. Can you check the stats of the files in a terminal?

**Comment by 5wattbulb** [comment]
Hi Jackson, the files did suddenly appear in VS Code so maybe it created them but I can't say. I can safely say that apart from Edge it was the only app I was running although I was running my own c app in manjaro linux terminal. Does this give any clues?

[tom@sp4 ~]$ ls -l
total 16
-rw-rw-rw-  1 tom tom    0 Mar 18 08:07 ''$'\001''CH'$'\200''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\r\025\367\362\a'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\r''0'$'\375\202''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 14:08 'A >'$'\200''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 14:01 'A'$'\224\365\202''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 'M'$'\327''Q'$'\202''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 13:54 'Q'$'\215\375\202''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ']M'$'\374\202''('$'\177'
drwxr-xr-x  8 tom tom 4096 Jan 31 09:53  dev
drwxr-xr-x  5 tom tom 4096 Aug 14  2021  dev3
drwxr-xr-x 17 tom tom 4096 Aug 14  2021  fossil
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 'm'$'\326''S'$'\200''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 08:05 'r'$'\230'' '$'\001'
drwxr-xr-x 16 tom tom 4096 Mar 23 19:09  sqlite
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 '}'$'\311\247\203''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\200'
-rw-rw-rw-  1 tom tom    0 Mar 18 14:07 ''$'\200'')'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\211\026'
-rw-rw-rw-  1 tom tom    0 Mar 18 08:06 ''$'\220'')'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\233\021'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\233\031'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\255\246''1'$'\200''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\257''%'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\273\036'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\300'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\300''!'
-rw-rw-rw-  1 tom tom    0 Mar 18 13:48 ''$'\301''Q'$'\374\202''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\302'\'''
-rw-rw-rw-  1 tom tom    0 Mar 18 14:07 ''$'\320'
-rw-rw-rw-  1 tom tom    0 Mar 18 13:45 ''$'\321\022\367\202''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\335''i'$'\375\202''('$'\177'
-rw-rw-rw-  1 tom tom    0 Mar 18 14:00 ''$'\340\036'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\350\t'
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\370'\'''
-rw-rw-rw-  1 tom tom    0 Mar 18 07:58 ''$'\375''L3'$'\200''('$'\177'
[tom@sp4 ~]$ stat ''$'\301''Q'$'\374\202''('$'\177'
  File: �Q��(
  Size: 0               Blocks: 0          IO Block: 4096   regular empty file
Device: 8,32    Inode: 131570      Links: 1
Access: (0666/-rw-rw-rw-)  Uid: ( 1001/     tom)   Gid: ( 1001/     tom)
Access: 2022-03-18 13:48:45.659253300 +0000
Modify: 2022-03-18 13:48:45.659253300 +0000
Change: 2022-03-18 13:48:45.659253300 +0000
 Birth: 2022-03-18 13:48:45.659253300 +0000
[tom@sp4 ~]$ stat ''$'\001''CH'$'\200''('$'\177'
  File: CH�(
  Size: 0               Blocks: 0          IO Block: 4096   regular empty file
Device: 8,32    Inode: 131562      Links: 1
Access: (0666/-rw-rw-rw-)  Uid: ( 1001/     tom)   Gid: ( 1001/     tom)
Access: 2022-03-18 08:07:06.069253300 +0000
Modify: 2022-03-18 08:07:06.069253300 +0000
Change: 2022-03-18 08:07:06.069253300 +0000
 Birth: 2022-03-18 08:07:06.069253300 +0000
[tom@sp4 ~]$

**Comment by JacksonKearl** [comment]
This looks very much like #145676, but I do not see issues deleting the files in that thread.

**Comment by 5wattbulb** [comment]
This is 145676 Jackson???

**Comment by hello-smile6** [comment]
> This looks very much like #145676, but I do not see issues deleting the files in that thread.

You just linked to `127.0.0.1`

**Comment by 5wattbulb** [comment]
If you look at the screenshot in my opening post you'll see there's an option to 'Create File' which I only just noticed myself. Today I tried this option for one of the offending files, added content to it, saved it and then tried deleting it. The deletion seemed to go smoothly (i.e.. no error message) but the filename just immediately pops back into place in vs code explorer and, when clicking on it, I'm back to 'File does not exist' square one.
